misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -4481,17 +4484,20 @@ private void preserveIndentation(boolean value) {
      */
     private int getPreservedIndentation(Token token) {
         int position = token.lineRange().startLine().offset();
+        int startLine = token.lineRange().startLine().line();
```

Comment:
Let's extract repeated call `token.lineRange().startLine()` to a variable say, `startLinePos`